src/core/mapping/base_mapper.cc

```diff
@@ -18,6 +18,7 @@
 #include <sstream>
 #include <unordered_map>
 
+#include "legion.h"
```

Comment:
I feel the existing headers already transitively include legion.h. did you actually need this to get the code compiled?